pages/docs/project-management/single-sign-on/okta-scim.mdx

```diff
@@ -0,0 +1,70 @@
+---
+description: Documentation for setting up Single Sign On with Okta
+---
+
+# Okta Directory Sync (SCIM)
+
+Follow these instructions to configure Okta attribute mappings so that you can control Hashboard roles through your default Okta profile. The procedure will be similar for other SCIM providers. Contact your support channel if you'd like assistance configuring SCIM for your organization.
```

Comment:
Should "support" here be a link to our support email? Or should we have a page in our docs on how you can contact us for support?